File: app/src/main/java/com/hotworx/ui/fragments/HotsquadList/CreateHotsquadFragment.java
package com.hotworx.ui.fragments.HotsquadList;

import android.annotation.SuppressLint;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ImageView;
import android.widget.RadioButton;
import android.widget.TextView;
import android.widget.Toast;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.core.content.ContextCompat;

import com.bumptech.glide.Glide;
import com.hotworx.R;
import com.hotworx.Singletons.ApiHeaderSingleton;
import com.hotworx.global.WebServiceConstants;
import com.hotworx.helpers.Utils;
import com.hotworx.models.LeaderBoard.Leaderboard;
import com.hotworx.requestEntity.LeaderBoardPOJO;
import com.hotworx.requestEntity.LeaderBoardResponse;
import com.hotworx.retrofit.GsonFactory;
import com.hotworx.ui.fragments.BaseFragment;
import com.hotworx.ui.fragments.LeaderBoard.LeaderGlobalFragment;
import com.hotworx.ui.views.TitleBar;

import java.util.ArrayList;

import butterknife.BindView;
import butterknife.ButterKnife;
import butterknife.OnClick;
import butterknife.Unbinder;

public class CreateHotsquadFragment extends BaseFragment {
    Unbinder unbinder;

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.fragment_hotsquad_create, container, false);
        unbinder = ButterKnife.bind(this, view);

        return view;
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
    }

    @Override
    public void setTitleBar(TitleBar titleBar) {
        super.setTitleBar(titleBar);
        titleBar.showBackButton();
        titleBar.setSubHeading(getString(R.string.leaderboard));
    }

    @Override
    public void onDestroy() {
        super.onDestroy();
      //  unbinder.unbind();
    }
}

File: app/src/main/java/com/hotworx/ui/fragments/HotsquadList/CreateHotsquadFragment.kt
package com.hotworx.ui.fragments.HotsquadList

import android.content.Intent
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.ProgressBar
import com.hotsquad.hotsquadlist.activity.LoginActivity
import com.hotworx.R
import com.hotworx.activities.MainActivity
import com.hotworx.databinding.FragmentHotsquadCreateBinding
import com.hotworx.helpers.Utils
import com.hotworx.helpers.UtilsHelpers
import com.hotworx.interfaces.LoadingListener
import com.hotworx.requestEntity.ExtendedBaseModel
import com.hotworx.retrofit.GsonFactory
import com.hotworx.ui.fragments.BaseFragment
import com.hotworx.ui.views.TitleBar
import okhttp3.ResponseBody
import retrofit2.Call
import retrofit2.Callback
import retrofit2.Response

class CreateHotsquadFragment : BaseFragment(){
    private var _binding: FragmentHotsquadCreateBinding? = null
    private val binding get() = _binding!!


    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        _binding = FragmentHotsquadCreateBinding.inflate(inflater, container, false)
        return binding.root
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
//        binding.btnCreateHotsquad.setOnClickListener{
//            val intent = Intent(requireContext(), LoginActivity::class.java)
//            startActivity(intent)
//        }
        binding.btnCreateHotsquad.setOnClickListener {
            val addListFragment = AddListFragment()
            dockActivity.replaceDockableFragment(addListFragment)
        }
    }

    override fun setTitleBar(titleBar: TitleBar) {
        super.setTitleBar(titleBar)
        titleBar.showBackButton()
        titleBar.subHeading = getString(R.string.leaderboard)
    }

    override fun onDestroyView() {
        super.onDestroyView()
        _binding = null
    }
}
